objects/b765cf/script.lua

```diff
@@ -144,13 +144,25 @@ function PostSetup(params)
 end
 
 function englandSnap(aidBoard)
-  local snapPoints = aidBoard.getSnapPoints()
-  local newSnapPoints = {}
-  for i,v in ipairs(snapPoints) do
-    if v.rotation.y <= 269 or v.rotation.y >= 271 then
-      table.insert(newSnapPoints,v)
+    do -- Remove aidBoard snap point
+        local snapPoints = aidBoard.getSnapPoints()
+        local newSnapPoints = {}
+        for i,v in ipairs(snapPoints) do
+            if table.concat(v.tags, "|") ~= "Invader Card" then
+                table.insert(newSnapPoints,v)
+            end
+        end
+        aidBoard.setSnapPoints(newSnapPoints)
+    end
+    do -- Add invader new discard snap point
+        local snapPoints = Global.getSnapPoints()
+        table.insert(snapPoints, {
+            position = Vector(-52.90, 1.3, -5.30),
```

Comment:
since this value is being used in 2 places now, would be nice to extract this at least to a local variable, if not a global one